### ISSUE TITLE
search filters in 1.5 dashboard, multiple keywords issues

### ISSUE THREAD

**Issue by bitsofinfo** [TykTechnologies/tyk#1432]
SEARCH filter in the new 1.5 dashboard dialogs. Seem like I can only search on one keyword

i.e. I have api names with both the terms "stage" and "rest" in them, but only one keyword seems relevant, specifying both return nothing. "stage" returns data, "rest" returns data, but "stage rest" returns nothing.

Its also unclear if I can use AND or OR terms between keywords (does not seem to work)

**Comment by nickReyn** [comment]
Thanks - yes searching functionality is very basic MVP at the mo. Will look to improve this functionality ASAP

**Comment by nickReyn** [comment]
@asoorm can you investigate whether it's possible to solve this using Mongo text search? :)

**Comment by bitsofinfo** [comment]
Any possibility simple "AND" support could be added when specifying multiple keywords, phrase support etc?

**Comment by buger** [comment]
Yes, coming in 2.6, full-text search. By default will be OR, in other words, multiple keywords will work, and phrases can be searched if wrapped in quotes.

**Comment by bitsofinfo** [comment]
not stale
